src/structures/breakend.cpp

```diff
@@ -0,0 +1,28 @@
+#include "structures/breakend.hpp"
+
+#include <cstdint>  // for enum struct _ : uint8_t
+
+enum struct sequence_type : uint8_t;
+
+enum struct strand : uint8_t;
```

Comment:
I don't understand these... are they enums of 256 values? Why are they not used below? Shouldn't they be declared in a header to be used elsewhere?